modules/rubiconBidAdapter.js

```diff
@@ -949,6 +882,63 @@ function addVideoParameters(data, bidRequest) {
   data.imp[0].video.h = size[1]
 }
 
+function applyFPD(fpd, mediaType, data) {
+  const map = {user: {banner: 'tg_v.', code: 'user'}, context: {banner: 'tg_i.', code: 'site'}, adserver: 'dfp_ad_unit_code'};
+  let obj = {};
+  let keywords = [];
+  const validate = function(e, t) {
+    if (typeof e === 'object' && !Array.isArray(e)) {
+      utils.logWarn('Rubicon: Filtered FPD key: ', t, ': Expected value to be string, integer, or an array of strings/ints');
+    } else if (e) {
```

Comment:
If the value passed in is falsey I think this will not do what we want.

For example if someone wanted to pass in the literal number zero for some reason `0`.